src/parser/add-hilites.js

```diff
@@ -2,6 +2,23 @@ const Hilites = require("../hilites")
 const Mark = require("mark.js")
 const IO = require("../util/io")
 
+/**
+ * Play a sound if a matched element has a sound attribute.
+ * @param {Element} ele Element which triggered the match.
+ */
+function play_sound(ele) {
+  const class_name = ele.className
+  const group = Hilites.Groups.get(null, {})[class_name]
+  if ("sound" in group) {
+    var audio = document.createElement("audio")
+    audio.style.display = "none"
+    audio.src = "file:///" + group.sound
+    audio.autoplay = true
+    audio.onended = () => audio.remove()
```

Comment:
this should probably also have `onerror`

Reply:
Added an `onerror` callback as well!